src/about/Intro.js

```diff
@@ -7,21 +7,76 @@ import urls from '../assets/urls'
+      <p>We believe Ethereum will one day be the the home of amazing dApps that are relied on by billions of users.</p>
+      <p>A decentralized version of Uber, for instance.</p>
+      <p>Yet Ethereum's ENTIRE NETWORK capacity right now is ~15 transactions per second.</p>
+      <p>Visa processes 45,000 transactions per second.</p>
+      <p>Supporting a decentralized Uber, Visa, AirBnB and others simply is not feasible at this time on Ethereum...</p>
+      <p>But there is an answer!</p>
+      <p>It's called "sharding".</p>
+      <p>The concept of sharding is based on splitting the entire state of the network into a bunch of partitions called shards that contain their own independent piece of state and transaction history. In this system, certain nodes would process transactions only for certain shards, allowing the throughput of transactions processed in total across all shards to be much higher than having a single shard do all the work as the mainchain does now.</p>
+      <p>You can read more about sharding in <a className="blue" href={urls.shardingExplained}>this blog post we put together</a>.</p>
+      <p>But don't leave! Bookmark that link, and keep reading on...</p>
+      <p>You see, sharding is one of, if not THE, most important thing for Ethereum's future.</p>
+      <p>And that's where Prysmatic Labs comes in.</p>
+      <p>Prysmatic Labs is a group of open-source contributors focused on making Sharding a reality. We're doing this by implementing the first <a className="blue" href={urls.sharding}>sharding</a> client for the go-ethereum project.</p>
+      <p>We are building out the first sharding implementation for the <a className="blue" href={urls.geth}> Geth (go-ethereum)</a> client, allowing the system to process transactions at over 10x - 100x of its current speed!</p>
+      <p><strong className="pink">But we need your help!</strong></p>
+      <p>Yodlers Yodl</p>
+      <p>Holders Hodl</p>
+      <p>Builders buidl...</p>
+      <p>But, those brave ones who seek to do what pundits have called impossible?</p>
+      <p>The crazy few who seek to scale Ethereum AND not do so via centralization?</p>
+      <p>Those, my dear friends, there rests a special spot in the lore of blockchain history.</p>
+      <p>Those crazy few are called "Shardlres".</p>
+      <p>And Shardlers #SHARDL</p>
+      <p>We know, we know... it's a title you'd love to wear proudly pretty much anywhere.</p>
+      <p>But guess what? Not only is there and opportunity to tell others you're a Shardler, the time is ripe where you can forever be known as one of the "Original Shardlers" who helped make Ethereum Sharding a reality and did so while starting the #SHARDL movement.</p>
+      <p>Yes, you can become one of the Original Shardlers who brought Sharding to Ethereum!</p>
+      <p>Here's how!</p>
```

Comment:
Hahah this is so cool! We can keep that terminology for the future :P, but for now let's not commit this to the site.